### PR TITLE
Calling API update

### PR DESCRIPTION
**Pull request by kanchm**
minor updates to the calling APIs to conform to Graph API standards

### CODE REVIEW

**Comment by jthake-msft** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Publish service&#39;s config file: .openpublishing.publish.config.json is not found. Exception: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811012057563658-3331%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Hb54jIhuo2hJp%2BN4n66I5DPstpeoE0vbje6IgGCqN5Y%3D&st=2018-11-01T20%3A53%3A14Z&se=2018-12-02T20%3A58%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/participant_configuremixer.md [hunk 1]
```diff
@@ -35,8 +35,9 @@ In the request body, provide a JSON object with the following parameters.
 
 ## Response
 Returns `202 Accepted` response code and a Location header with a uri to the [commsOperation](../resources/commsoperation.md) created for this request.
+
 ## Example
-The following example shows how to call this API.
+Here is an example of how to call this API.
```
Comment (Lauragra) [inline api-reference/beta/api/participant_configuremixer.md]: Please restore the original text.

**Comment by jthake-msft** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Publish service&#39;s config file: .openpublishing.publish.config.json is not found. Exception: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811012128439481-3331%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ueKw2ijxgT73LEgkoymaWoLOv2pqnJo5rOODabFp0Nw%3D&st=2018-11-01T21%3A24%3A04Z&se=2018-12-02T21%3A29%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/participant_invite.md [hunk 2]
```diff
@@ -37,7 +37,8 @@ In the request body, provide a JSON object with the following parameters.
 Returns `202 Accepted` response code and a Location header with a uri to the [commsOperation](../resources/commsoperation.md) created for this request.
 
 ## Example
-The following example shows how to call this API.
+Here is an example of how to call this API.
```
Comment (Lauragra) [inline api-reference/beta/api/participant_invite.md]: Restore original text.

File: api-reference/beta/api/participant_invite.md [hunk 3]
```diff
@@ -224,13 +231,14 @@ Content-Type: application/json
   ]
 }
 ```
+
```
Comment (Lauragra) [inline api-reference/beta/api/participant_invite.md]: FYI, a topic can only have one H1 heading, which is the title. Please review the structure of this topic and ensure that there is only one H1. We should have one H2 section called Examples, and each example is an H3 section within it, and H4s/H5s within as needed.

File: api-reference/beta/api/participant_mute.md [hunk 4]
```diff
@@ -33,10 +33,10 @@ In the request body, provide a JSON object with the following parameters.
 |clientContext|String|The client context.|
 
 ## Response
-If successful, this method returns `200, OK` response code and [commsOperation](../resources/commsoperation.md) object in the response body.
+If successful, this method returns `200 OK` response code and [commsOperation](../resources/commsoperation.md) object in the response body.
 
 ## Example
-The following example shows how to call this API.
+Here is an example of how to call this API.
```
Comment (Lauragra) [inline api-reference/beta/api/participant_mute.md]: Restore original.

File: api-reference/beta/resources/call.md [hunk 5]
```diff
@@ -41,26 +41,26 @@ Although the same identity cannot be invited multiple times, it is possible for
...
 | mediaConfig         | [appHostedMediaConfig](apphostedmediaconfig.md) or [serviceHostedMediaConfig](servicehostedmediaconfig.md) | The media configuration.                                                                                                                                                                        |
-| meetingCapability   | [meetingCapability](meetingcapability.md)                                                              | Contains the capabilities of a meeting                                                                                                                                                                                                    |
+| meetingCapability   | [meetingCapability](meetingcapability.md)                                                              | Contains the capabilities of a meeting                                                                                                                                                              |
```
Comment (Lauragra) [inline api-reference/beta/resources/call.md]: Please consistently add periods at the end of each description.

**Comment by jthake-msft** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Publish service&#39;s config file: .openpublishing.publish.config.json is not found. Exception: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811012141261189-3331%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Y%2B1Z%2FtBUtG3Y6tHqbqGL76KSN%2Fm%2FfAx9qTOzeQ1iiG4%3D&st=2018-11-01T21%3A36%3A47Z&se=2018-12-02T21%3A41%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Publish service&#39;s config file: .openpublishing.publish.config.json is not found. Exception: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811012243228912-3331%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FC5Hbsu222TWyqxzHvtveJp5FSsl9fjja%2F%2FF%2BzeKc1Q%3D&st=2018-11-01T22%3A38%3A42Z&se=2018-12-02T22%3A43%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/participant_invite.md (hunk 3 above)
Comment (kanchm) [reply api-reference/beta/api/participant_invite.md]:
```raw
# participant: invite
## Permissions
## HTTP request
## Request headers
## Request body
## Response
## Examples
##### Request
##### Response
### Invite Participants in Existing P2P meeting
##### Request
##### Response
##### Notification - operation completed
##### Notification - roster updated with participant added
### Invite Participants in Existing P2P meeting
##### Answer Incoming VOIP call with service hosted media
##### Notification - Incoming
##### Request
##### Response
##### Notification - Establishing
##### Notification - Established
### Join channel meeting without media
##### Request
##### Response
##### Notification - Establishing
##### Notification - Established
### Invite participant from initial incoming call
##### Response
##### Notification - Operation Completed
##### Notification - Roster Updated With Participant Added
##### Notification - terminated the original P2P call
```

Comment (kanchm) [reply api-reference/beta/api/participant_invite.md]: Done

File: api-reference/beta/resources/call.md (hunk 5 above)
Comment (kanchm) [reply api-reference/beta/resources/call.md]: Done

File: api-reference/beta/api/participant_configuremixer.md (hunk 1 above)
Comment (kanchm) [reply api-reference/beta/api/participant_configuremixer.md]: Done

File: api-reference/beta/api/call_updatemetadata.md [hunk 6]
```diff
@@ -37,7 +37,7 @@ In the request body, provide a JSON object with the following parameters.
 Returns `202 Accepted` response code and a Location header with a uri to the [commsOperation](../resources/commsoperation.md) created for this request.
 
 ## Example
-The following example shows how to call this API.
+Here is an example of how to call this API.
```
[the remark this answers is not in the document]
Comment (kanchm) [reply api-reference/beta/api/call_updatemetadata.md]: Done

**Comment by jthake-msft** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Publish service&#39;s config file: .openpublishing.publish.config.json is not found. Exception: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811020350371090-3331%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=XrBu52MaNHuJulw0Ux6yoeQJwoqfYoCHtKwnoNIpDO8%3D&st=2018-11-02T03%3A45%3A54Z&se=2018-12-03T03%3A50%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kanchm** [comment]
@Lauragra I am getting this error which does not seem related to my changes.

```code
    MSBUILD : error : Error 11/1/2018 8:58:13 PM: Publish service's config file: .openpublishing.publish.config.json is not found. Exception: . 
    Verbose 11/1/2018 8:58:13 PM: Step 'SyncGitSourceWithDownloadingPackage' finished in '13802.5664' ms.. 
Done building target "SyncGitSourceWithDownloadingPackage" in project "publish-local.targets" -- FAILED.
```

I also tried to look for more details in the [OSP portal](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=pullrequests) but I see others also hitting this error.  Is the branch broken?

**Comment by Lauragra** [comment]
@Jeremy Thake<mailto:Jeremy.Thake@microsoft.com> is working on that issue, should be cleared up soon.

-Laura

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by jthake-msft** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Publish service&#39;s config file: .openpublishing.publish.config.json is not found. Exception: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811020703598986-3331%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dpNKCccOtBb73Q0UrdYsYGsZqrcZZ%2BEbJT5Eoq9Hu1s%3D&st=2018-11-02T06%3A59%3A18Z&se=2018-12-03T07%3A04%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Publish service&#39;s config file: .openpublishing.publish.config.json is not found. Exception: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C3%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811030041565036-3331%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=U5RPXFWeYjegJiNUpiEFpcvPBcTMi00rNNzzqdJa2q0%3D&st=2018-11-03T00%3A37%3A40Z&se=2018-12-04T00%3A42%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kanchm** [comment]
@Lauragra please let us know if anything else I would need to change to push this through. We are releasing our next version of the SDK and it would be good to have this documentation go out together with the SDK.

**Comment by Lauragra** [comment]
@kanchm , I'll get this in with tonight's publish if you're ready.

File: api-reference/beta/resources/calls-api-overview.md [hunk 7]
```diff
@@ -83,9 +83,8 @@ The Media SDK allows the bot to send and receive audio, video, and video-based s
...
 
-## Known issues
+## See also
 
-The following are known issues with the calls and online meetings API:
+[Calls and online meetings API samples](https://github.com/microsoftgraph/microsoft-graph-comms-samples/)
 
-- Case mismatch in payloads - Payloads returned in notifications and responses contain incorrect capitalization of odata.types. They are returned in upper camel case instead of lower camel case.
-- Navigation path `/applications/{id}` is not supported - Navigating through the global applications node to the application, even your own, is not allowed. Please use the `/app` navigation only.
+[Known issues](../../../concepts/known_issues.md#Calls-and-online-meetings)
```
[the remark this answers is not in the document]
Comment (Lauragra) [reply api-reference/beta/resources/calls-api-overview.md]: This link should be okay. It's images that we link to the CDN for.

**Comment by kanchm** [comment]
@Lauragra thanks for pushing this through. These are our active APIs and getting them out soon is preferred.

**Comment by Lauragra** [comment]
We have a merge conflict and I’m unable to check in the fix. Will have to look into resolving on Monday.

-Laura

**Comment by Lauragra** [comment]
Yes, the issue is a merge conflict with the master branch. We’ll work it out on Monday - thanks!

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Pull request merged**